doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -131,6 +131,17 @@ the most recent status for each AWS PrivateLink connection in the system.
 | `last_status_change_at` | [`timestamp with time zone`] | Wall-clock timestamp of the connection status change.|
 | `status` | [`text`] | | The status of the connection: one of `pending-service-discovery`, `creating-endpoint`, `recreating-endpoint`, `updating-endpoint`, `available`, `deleted`, `deleting`, `expired`, `failed`, `pending`, `pending-acceptance`, `rejected`, or `unknown`. |
 
+## `mz_cluster_blue_green_lineage`
```

Comment:
Can we call this "deployment" instead of "blue/green"? I think that's a bit cleaner/clearer!

```suggestion
## `mz_cluster_deployment_lineage`
```